src/pages/blog/index.tsx

```diff
@@ -60,11 +61,89 @@ const filterByTag: (entry: IBlogEntry, tagQuery?: string | null) => boolean = (
 const Blog: React.FC<BlogProps> = ({ blogs, tags }) => {
   const [searchQuery, setSearchQuery] = useState<string>('');
   const [hash, setHash] = useHash({ shallow: false });
+  const { query, isReady } = useRouter();
+  const { page, search } = query;
+
+  useEffect(() => {
```

Comment:
We won't get any valid data until `isReady` is set to true, would this make sense?
```suggestion
  useEffect(() => {
  if (!isReady) return;
```

Reply:
Is replaced with the useOnce hook, so shouldnt be necessary anymore.